lib/ansible/modules/cloud/amazon/aws_glacier_vault.py

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/python
+#
+# This is a free software: you can redistribute it and/or modify
```

Comment:
Use the newer simple copyright form: http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#copyright